default.json

```diff
@@ -364,6 +374,16 @@
         "rancher/helm"
       ]
     },
+    {
+      "description": "Disable digest updates (commit-based versions) for Go modules",
+      "matchManagers": [
+        "gomod"
+      ],
+      "matchUpdateTypes": [
+        "digest"
+      ],
+      "enabled": false
```

Comment:
Given that we are actively disabling this, we should explicitly enable the next rule.